README.md

```diff
@@ -105,19 +105,6 @@ ApmCollector::startMeasure('my-custom-span', 'custom', 'measure', 'My custom spa
 ApmCollector::stopMeasure('my-custom-span');
 ```
 
-To record an additional span around your job execution, you may include the provided job middleware (Laravel 6+ only https://laravel.com/docs/6.x/queues#job-middleware):
```

Comment:
Could you also update the supported Laravel version? It says now 5.5+ which is not really accurate. There's also a mention to Laravel 7, it should be removed as well.

Reply:
Done.